src/corner-dialog/src/CornerDialog.js

```diff
@@ -39,9 +39,7 @@ const closeAnimation = css.keyframes('closeAnimation', {
 
 const animationStyles = {
   '&[data-state="entering"], &[data-state="entered"]': {
-    animation: `${openAnimation} ${ANIMATION_DURATION}ms ${
-      animationEasing.spring
-    } both`
+    animation: `${openAnimation} ${ANIMATION_DURATION}ms ${animationEasing.spring} both`
```

Comment:
Prettier auto-formatted this for me, I can change it back if we don't want any unrelated diffs :)